.github/scripts/build_aiopnsense_release_notes.py

```
@@ -0,0 +1,300 @@
+    logging.basicConfig(level=logging.INFO, format="%(levelname)s: %(message)s")
+    args = _parse_args(sys.argv[1:] if argv is None else argv)
+    token = os.environ.get("GITHUB_TOKEN")
+    try:
+        releases = fetch_releases(owner=args.release_owner, repo=args.release_repo, token=token)
+    except HTTPError as err:
+        LOGGER.error(
+            "Failed to fetch releases for %s/%s: %s",
+            args.release_owner,
+            args.release_repo,
+            err,
+        )
```

Ruff (0.15.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)